packages/aws-cdk/lib/cli.ts

```diff
@@ -556,15 +575,22 @@ function yargsNegativeAlias<T extends { [x in S | L ]: boolean | undefined }, S
   };
 }
 
+let SHOW_ALL_NOTICES = false;
 export function cli() {
   initCommandLine()
-    .then(value => {
-      if (value == null) { return; }
+    .then(async ({ value, configuration }) => {
       if (typeof value === 'string') {
         data(value);
       } else if (typeof value === 'number') {
         process.exitCode = value;
       }
+      await displayNotices({
```

Comment:
I would also like a `refreshNotices();` function called at the start of the CLI, just to be able to do some latency masking.